### ISSUE TITLE
What is the loss function in sequencetagger when training the model with crf?

### ISSUE THREAD

**Issue by mittalsarthak** [flairNLP/flair#759]
What is the loss of the sequence tagger model in case we are training the model with CRF? Like when we are not using CRF the loss function is cross entropy. But in the case of CRF, it is calculating 2 values (i.e forward score and gold score). What are these scores?

**Comment by Trsten** [comment]
What I found in code:
# the loss function is Viterbi if using CRF, else regular Cross Entropy Loss
https://github.com/flairNLP/flair/blob/master/flair/models/sequence_tagger_model.py
